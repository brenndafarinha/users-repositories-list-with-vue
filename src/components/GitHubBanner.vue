<template>
  <v-container>
    <v-row align="center" justify="center" class="d-flex flex-column justify-center mt-10">
      <v-img src="../assets/GitHub_Logo.png" width="600px" />
      <v-col class="text-center">
        <h1 class="display-1">Lista de reposit√≥rios</h1>
      </v-col>
    </v-row>
  </v-container>
</template>
<script>
export default {
  name: "GitHubBanner"
};
</script>