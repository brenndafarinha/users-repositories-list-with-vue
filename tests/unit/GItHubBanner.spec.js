
import { shallowMount } from '@vue/test-utils';
import GitHubBanner from '../../src/components/GitHubBanner';

describe('GitHubBanner', () => {
    it("renders correctly", () => {
        const wrapper = shallowMount(GitHubBanner);
        expect(wrapper).toMatchSnapshot();
      });
});