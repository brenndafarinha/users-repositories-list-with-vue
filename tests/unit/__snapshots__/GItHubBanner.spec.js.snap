// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubBanner renders correctly 1`] = `
<v-container>
  <v-row align="center" justify="center" class="d-flex flex-column justify-center mt-10">
    <v-img src="../assets/GitHub_Logo.png" width="600px"></v-img>
    <v-col class="text-center">
      <h1 class="display-1">Lista de reposit√≥rios</h1>
    </v-col>
  </v-row>
</v-container>
`;
